hi_markt_webcam.py

```diff
@@ -18,13 +20,17 @@
     # assemble wget command 
     s_wget_command = f"wget -O {s_filename} --user-agent='Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0' {s_download_url}"
 
+    # delay execution by random amount to hopefully prevent overBoting the server and getting to much 404s
```

Comment:
In software development, there is no '**hope**'  ;)

Reply:
Also the issue of 404s is not related to the requests made from our client, rather an issue on server side or just to much requests in general. 
So nothing we can do here.